uaclient/messages.py

```diff
@@ -1029,3 +1021,8 @@ class TxtColor:
 with '{{service}}' enabled""".format(
     bold=TxtColor.BOLD, end_bold=TxtColor.ENDC
 )
+
+ENTITLEMENT_NOT_FOUND = FormattedNamedMessage(
```

Comment:
I am wondering if we should use this exception on the `action_enable` function on `cli`

Reply:
`action_enable` currently has a more descriptive error message for a human at the terminal in the case where it is passed an unknown entitlement. Maybe we should revisit that message and consolidate, but I'd like to consider it out of scope for this PR.